Podcast/OnboardingViewController.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  OnboardingViewController.swift
```

Comment:
In future, we will use UIPageControl and UIScrollView

Reply:
agreed .. talked about this offline